build.rs

```diff
@@ -0,0 +1,7 @@
+use std::process::Command;
+fn main() {
+    Command::new("pip")
+        .args(vec!["install", "-r", "requirements.txt"])
```

Comment:
I would add the `--user` flag so that we only install it for the current user (so we don't need root privileges)